nix/nixpkgs.nix

```diff
@@ -27,7 +41,7 @@ let
         # and/or documentation.
         (pkgs.fetchurl {
           url = "https://patch-diff.githubusercontent.com/raw/bazelbuild/bazel/pull/8983.patch";
-          sha256 = "1j25bycn9q7536ab3ln6yi6zpzv2b25fwdyxbgnalkpl2dz9idb7";
+          sha256 = "1qdyqymsylinkdwqhbxbm8bvbyznrdn74n744pi5xhdwb6lw1r8a";
```

Comment:
this is unrelated, changing the JVM forced a rebuild and needed a hash change because this file has changed upstream. There don’t seem to be any relevant changes and the PR has now been closed (looks like the change has made it into Bazel 4.0) so hopefully it will stay stable until we get to Bazel 4.0 and can drop it.